src/librustc_trait_selection/traits/wf.rs

```diff
@@ -34,6 +34,22 @@ pub fn obligations<'a, 'tcx>(
     }
 }
 
+/// Returns the set of obligations needed to make `c` well-formed.
```

Comment:
This says `c` but the code uses `constant`. If you want to be short, you can also use `ct`.

Reply:
forgot to update the comment